tests/form/robot/ValidationTextBox.html

```diff
@@ -495,13 +495,39 @@
 							doh.robot.typeKeys('123', 1000, 600);
 							doh.robot.sequence(d.getTestCallback(dojo.hitch(this, function(){
 								doh.is('123', this.textbox.focusNode.value, "focusNode.value");
-								doh.is(undefined, this.textbox.get('value'), "get('value')");
+								doh.is(123, this.textbox.get('value'), "get('value')");
 								doh.f(this.textbox.isValid(), "!isValid()");
```

Comment:
Hmm, maybe it is correct for `isValid()` to return until the blur.  Relatedly, presumably before the blur, `ValidationTextBox#state` is "Incomplete" rather than "Normal"?  But why does `get("value")` return a valid value at that point?   That's the part that seems inconsistent.

On a more minor point, can't the "missing required decimal" and "missing required decimal, format on blur" test be combined into one test?   Type "123", check conditions, then blur, then check conditions again.